### DISCUSSION TITLE
False Positive for table extraction

### DISCUSSION THREAD

**Discussion by pranjal-jaiswal** [jsvine/pdfplumber#524] [Ask for help with specific PDFs]
I am getting a table object on the first page of a pdf and there is not a single vertical or horizontal line on 1st page.

I have attached the code snippet and result for your reference:

Code Snippet:
import pdfplumber
my_pdf = pdfplumber.open("2.pdf")
tables = my_pdf.pages[0].find_tables()
tables[0].extract()

Result:
[["Here's my SSN: 608-18-8954. I have been routing money 6227-8231-4. Ben is from"],
 ['Action Auto. congratulations on passing your driving test 6940579.. need to see last'],
 ['10 transaction of card 4539254270870707. male social security number is'],
 ['768-52-0198. Borbála is very reliable. You can always depend on him.. can you tell'],
 ['me your medical license number? QO5772175.. Zoolander is a 2001 American'],
 ['action-comedy film directed by Apolonia Kamińska and starring Kamińska. my name'],
 ['on the bank account 380106394 is wrong. what do i have to do to get it corrected?'],
 ['']]

I have also attached the pdf for reference:
[2.pdf](https://github.com/jsvine/pdfplumber/files/7425811/2.pdf)

**Comment by samkit-jain** [comment]
Hi @pranjal-jaiswal Appreciate your interest in the library. If you use the table debugger, you'll notice that there are hidden horizontal and vertical lines as you can see in the below output.
![icici](https://user-images.githubusercontent.com/15127115/139076070-034bf73a-1e7f-48a3-a47b-0763a79e83a9.png)

To remove them, you can refer to the code at https://github.com/jsvine/pdfplumber/issues/285#issuecomment-710111180

**Reply by pranjal-jaiswal** [reply]
Hi @samkit-jain, Thanks for the prompt reply, I will give it a try
